### DISCUSSION TITLE
What contributes to Movies/TV Series that sghow up on the carious pages

### DISCUSSION THREAD

**Discussion by mdwkelly** [seerr-team/seerr#413] [Q&A] [answered]
Hi,

Looking to move over from Ombi to Jellyseerr and am wondering what contributes to the movies that show up in the vairous pages, obviously not the ones from my collection or the requested ones, but the other one that are pulled in based upon something in my setup.

For example, I have some foreign, mostly Asian, movies my my collection and a lot of the other movies being pulled in are foreign, Asian, but are on the adult side of the fence and not something I want to be showing, or available to request, for my users. I will say that I have watched a couple of them in the past through Emby but they are no longer in my collection.

Thanks for any info

**Comment by fallenbagel** [comment]
![Screenshot_20230623_094938_Chrome Dev.jpg](https://github.com/Fallenbagel/jellyseerr/assets/98979876/2592cdc0-ab97-408e-b286-1e21b15db51f)

Do you mean this page?

**Reply by mdwkelly** [reply]
Yes, that one.

Scrolling through there are a number of movies appearing that could be requested where the covers/titles are not appropriate for family or kids

Checking the details, many are tagged as "pink movie" but others are not.

Thanks

**Reply by fallenbagel** [reply, marked as answer]
It's just the movie api endpoint from tmdb. You could filter or better yet set discover region/language to something specific.

The reason for the "pinku" movies and stuff is a limitation of tmdb. Jellyseerr already filters adult content but there are some from tmdb that is not marked as adult which slips through and is what you see. To fix this from the source itself, you can edit it in tmdb and set it to adult manually from themoviedb.org as it's opensource.

**Reply by mdwkelly** [reply]
Thank you for the quick reply and for the information, appreciated. I will do some trial and error but at least I understand why it is happening.

Been on jellyseerr for only a couple of days and it looks great. Thank you for all your hard work.

Cheers

**Reply by fallenbagel** [reply]
It's thanks to the hardwork from the team at overseerr c:

**Comment by mdwkelly** [comment]
Very true.

The fork and changes are no small feat either.

Cheers
